.gitignore

```diff
@@ -416,3 +416,4 @@ utils/DXFiles/
 !*.dll
 !*.exe
 utils/vswhere.exe
+refactor/
```

Comment:
Whoever merges this PR, dont forget to remove this.

Reply:
https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally

Reply:
thanks